src/Client.cpp

```diff
@@ -874,15 +874,24 @@ void CClient::HandleCap(const CString& sLine)
 	//TODO support ~ and = modifiers
 	CString sSubCmd = sLine.Token(1);
 
+	std::map<CString, std::function<void(bool bVal)>> mCoreCaps = {
```

Comment:
I just wanted to decrease code duplication... Now this list is listed 4 times (including CLEAR)